content/docs/start/model-management/index.md

```diff
@@ -9,23 +9,31 @@ description:
 
 # Get Started: Model Management
 
+https://www.youtube.com/watch?v=T7MBFpnSr9Q
+
 ## Chapters
 
-- **[Model registry]** - Set up a git-based model registry with DVC to track and
-  manage models, their versions and lifecycle stages.
+- **[Add a model]** - Start tracking model artifacts with DVC.
+
+- **[Model registry]** - Manage models, their versions and lifecycle stages in a
+  git-based model registry.
 
 - **[Using and deploying models]** - Easily download your models from the model
   registry. Set up your CICD pipelines to be trigger by model registry actions
   (such as assigning model stages) and deploy models directly form the model
   registry.
 
+[Add a model]: /doc/start/model-management/add-model
 [model registry]: /doc/start/model-management/model-registry
 [Using and deploying models]: /doc/start/model-management/model-cicd
 
-<admon type="tip">
+<admon type="info">
 
-These are captured in our [example-get-started-experiments] repo (you can [fork
-it][example-get-started-experiments-fork] to follow along).
+These are captured in our [example-get-started-experiments] repo. You can [fork
+it][example-get-started-experiments-fork] to follow along, or you can take a
+look at our
+[public model registry](https://studio.iterative.ai/team/Iterative/models) (read
```

Comment:
Add link to public model registry for those who just want to see what it looks like